js/content/community.js

```diff
@@ -596,14 +581,33 @@ let ProfileHomePageClass = (function(){
             }
         }
 
-        if (SyncedStorage.get("profile_permalink")) {
-            document.querySelector("#es_permalink").addEventListener("click", function(e) {
-                e.target.select();
-            });
-            document.querySelector("#es_permalink_copy").addEventListener("click", function(e) {
-                document.querySelector("#es_permalink").select();
-                document.execCommand('copy');
-            });
+        function copySteamId(e) {
+            let elem = e.target.closest(".es_copy");
+            if (!elem) { return }
+
+            let text = elem.dataset.copy;
+            Clipboard.set(text);
+
+            ExtensionLayer.runInPageContext(`function() {
+                window.idDialog.Dismiss();
+                ShowAlertDialog(g_rgProfileData.personaname + "'s SteamID", \`${Localization.str.copied}\`.replace("__text__", "${text}"));
```

Comment:
It's annoying to have another confirmation window popping up, I'd completely remove it

Reply:
Sorry, I meant dialog.
Could you nevertheless please remove it?

Reply:
Done while maintaining a 'copied' visual notification